qiskit_experiments/framework/composite/batch_experiment.py

```diff
@@ -69,17 +86,28 @@ def circuits(self):
                 qubit_mapping = None
             else:
                 qubit_mapping = [self._qubit_map[qubit] for qubit in expr.physical_qubits]
-            for circuit in expr.circuits():
-                # Update metadata
-                circuit.metadata = {
-                    "experiment_type": self._type,
-                    "composite_metadata": [circuit.metadata],
-                    "composite_index": [index],
-                }
-                # Remap qubits if required
-                if qubit_mapping:
-                    circuit = self._remap_qubits(circuit, qubit_mapping)
-                batch_circuits.append(circuit)
+
+            if isinstance(expr, BatchExperiment):
+                circ_groups = expr._batch_circuits()
+            else:
+                circ_groups = [(expr.circuits(), expr)]
+
+            for group in circ_groups:
+                local_circuits = []
```

Comment:
Local doesn't make physical sense to me. Since this is a batch experiment, we are just copy circuit from standard experiment to batch circuit without any composition or fusion. I see this is somewhat of chunk rather than local/global.